spring-cloud-gateway-core/src/test/java/org/springframework/cloud/gateway/route/builder/RouteBuilderTests.java

```diff
@@ -63,6 +68,31 @@ public void testASetOfRoutes() {
 				.expectComplete().verify();
 	}
 
+	@Test
+	public void testRoutesWithTimeout() {
```

Comment:
so what does this test actually tests? :) can we give it some reasonable name?

Reply:
I tried to keep the same tests name style as already exists in this class